docs/_static/js/script.js

```diff
@@ -23,13 +23,15 @@ $(function() {
 
     /* Collapse specified sections in the installation guide */
     if(window.location.pathname.toLocaleLowerCase().indexOf('installation-guide') != -1) {
-        $('<style>.closed, .opened {cursor: pointer;} .closed:before, .opened:before {font-family: FontAwesome; display: inline-block; padding-right: 6px;} .closed:before {content: "\\f078";} .opened:before {content: "\\f077";}</style>').appendTo('body');
+        $('<style>.closed, .opened {cursor: pointer;} .closed:before, .opened:before {font-family: FontAwesome; display: inline-block; padding-right: 6px;} .closed:before {content: "\\f054";} .opened:before {content: "\\f078";}</style>').appendTo('body');
```

Comment:
I think that [`chevron-right: >`](https://fontawesome.com/icons/chevron-right?f=classic&s=solid) for closed section and [`chevron-down: v`](https://fontawesome.com/icons/chevron-down?f=classic&s=solid) for opened sections are more intuitive.